tests/lib/nested.sh

```diff
@@ -795,6 +795,27 @@ copy_remote(){
     sshpass -p ubuntu scp -P "$SSH_PORT" -o ConnectTimeout=10 -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no "$@" user1@localhost:~
 }
 
+run_local_test(){
```

Comment:
`local` here isn't quite right, it's more that this is running an external test on a nested system, right? if so I would rename this as

```suggestion
run_external_test_on_nested_system(){
```